distribution/ddf/src/main/filtered-resources/etc/org.apache.karaf.features.cfg

```diff
@@ -30,6 +30,7 @@ featuresRepositories= \
  mvn:org.codice.opendj.embedded/opendj-embedded-app/${opendj-embedded.app.version}/xml/features, \
  mvn:org.codice.ddf.spatial/spatial-app/${project.version}/xml/features, \
  mvn:ddf.ui.search/search-ui-app/${project.version}/xml/features, \
+ mvn:ddf.ui/catalog-ui-app/${project.version}/xml/features, \
```

Comment:
This will need to be removed if/when we merge the search and catalog UI feature files/apps.